config/initializers/1_fast_gettext.rb

```diff
@@ -15,6 +15,11 @@
 # work in all domains context by default (for plugins)
 include FastGettext::TranslationMultidomain
 
+# Keep TRANSLATORS comments
+Rails.application.config.gettext_i18n_rails.xgettext = %w[--add-comments=TRANSLATORS:]
+# Disable fuzzy .po merging
+Rails.application.config.gettext_i18n_rails.msgmerge = %w[--no-fuzzy-matching]
```

Comment:
I'll test plugins, but since it's in Foreman's initialiser, I'd have thought they'd inherit it naturally?